File: Basic/Compute Length of A String/SolutionbyArchit.cpp
#include<bits/stdc++.h>
using namespace std;


//Program to compute length of string by Archit Garg CSE 1st year


int find_length(string str){
  int length=0;

 for(int i=0;str[i]!='\0';i++) 
 ++length;

 
  return length;
}


int main()
{
    string str;
    getline(cin,str);

    cout<<"length = "<<find_length(str);  
   

}


      
    

File: Data Structures/Array Or Vector/Product of Elements in An Array/SolutionByArchit.java
import java.io.*;
import java.util.*;

public class solution {
    // program to compute product of all the elements by Archit garg CSE 1st year

    public static int product(int[] arr,int idx){
        if(idx==arr.length){
            return 1;
        }

        return arr[idx]*product(arr,idx+1);
    }

    public static void main(String[] args) {
       
        Scanner scn=new Scanner(System.in);
        System.out.println("Enter size");
        int n = scn.nextInt();

        int[] arr=new int[n];
        System.out.println("Enter array");
        for(int i=0;i<n;i++)
            arr[i]=scn.nextInt();

            System.out.println(product(arr,0));
        
      
        }
}
